package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "sentry-graph-renderer",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "bin": {
+    "chart-renderer": "./lib/chartRenderer.js"
+  },
+  "scripts": {
+    "lint": "tsc && eslint src/**.ts",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
+  "dependencies": {
+    "@sentry/node": "^6.2.0",
+    "@types/echarts": "^4.9.3",
+    "@types/express": "^4.17.11",
+    "@types/yargs": "^16.0.0",
+    "canvas": "^2.6.1",
+    "dotenv": "^8.2.0",
+    "echarts": "4.7.0",
+    "express": "^4.17.1",
+    "webpack-hook-plugin": "^1.0.7",
+    "yargs": "^16.2.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.13.0",
+    "@babel/plugin-transform-runtime": "^7.13.6",
+    "@babel/preset-env": "^7.13.5",
+    "@babel/preset-typescript": "^7.13.0",
+    "babel-loader": "^8.2.2",
+    "eslint": "5.11.1",
+    "eslint-config-sentry-app": "1.50.0",
+    "file-loader": "^6.2.0",
+    "fork-ts-checker-webpack-plugin": "^6.1.0",
+    "jest": "^26.6.3",
+    "null-loader": "^4.0.1",
+    "prettier": "^2.2.1",
+    "ts-node": "^9.1.1",
```

Comment:
Is this still used?

Reply:
It was used to transpire webpack config. But after that’s gone it will be gone too